docs/cli.md

```diff
@@ -139,6 +139,11 @@ The environment variables listed below take precedence over the DataHub CLI conf
 - `ACTIONS_VERSION` (default `head`) - Set to a specific version to run quickstart with that image tag of `datahub-actions` container.
 - `DATAHUB_ACTIONS_IMAGE` (default `acryldata/datahub-actions`) - Set to `-slim` to run a slimmer actions container without pyspark/deequ features.
 
+For images `acryldata/datahub-ingestion` or `acryldata/datahub-ingestion-slim`
```

Comment:
the image is `acryldata/datahub-ingestion:<whatever>-slim` right?

but then we still have `acryldata/datahub-actions-slim`?

Reply:
Forgot to update this after that change. There is no longer a `acryldata/datahub-actions-slim`, there is just the `-slim` tag as you said. Will update, thanks!

Reply:
Actually this part will be removed per the next comment.